README.md

```diff
@@ -30,7 +29,7 @@ data Options
   }
   deriving (Show, GHC.Generics.Generic)
 
-instance Generics.SOP.Generic Options
+instance System.Console.GetOpt.Generics.Generic Options
```

Comment:
All LGTM.

I think it's inconvenient that we have two typeclasses called `Generic` and users need to reference both. So at least one of them has to be referenced qualified. I think the best way to use this would be to import `GHC.Generics` qualified and then refer to `Generics.SOP.Generic` unqualified. But I didn't want to do that in the README because I wanted it to be very clear. Any remarks on that?